### DISCUSSION TITLE
Setup Chai Linter

### DISCUSSION THREAD

**Discussion by kibettheophilus** [droidconKE/chai#62] [General]
Hey @tamzi , I see that there hasn't been much activity on [chailinter](https://github.com/droidconKE/chai/tree/main/chailinter). I am interested in picking up the setup for the linter if there are plans to still be worked on.

PS: I gave a talk about [Creatiing Custom Lints](https://speakerdeck.com/kibet/crafting-custom-lint-rules-elevating-android-code-quality) 😊

**Comment by tamzi** [comment]
Create branch and let us do it @kibettheophilus 

Lots of refactoring happening.

**Reply by kibettheophilus** [reply]
@tamzi sawa
